pkg/events/parsers/go.mod

```diff
@@ -0,0 +1,22 @@
+module tracee/pkg/events/parsers
+
+go 1.21.6
```

Comment:
For this let's try to stick with the minimum version as 1.21 as the main go.mod and do not insert the toolchain since it will be imported by other consumers.